portal/models/qb_timeline.py

```diff
@@ -559,7 +570,13 @@ def ordered_qbs(user, classification=None):
                         if q.instrument not in common:
                             continue
                         qnr = QuestionnaireResponse.query.get(q.qnr_id)
-                        qnr.qb_id = rp_flyweight.nxt_qbd.qb_id
+                        assert qnr.authored >= rp_flyweight.nxt_start
+                        # when RP dates don't align, attempt an advance
+                        if qnr.authored > rp_flyweight.nxt_exp:
+                            rp_flyweight._advance_nxt()
+                            if qnr.authored > rp_flyweight.nxt_exp:
+                                raise ValueError(f"transition error on qnr {qnr.id}")
```

Comment:
@pbugni  looks like a syntax error here

Reply:
how so?

Reply:
the double check is intentional - might need to advance one.  if that doesn't catch us up, we're in an error state.

Reply:
Sorry, I likely misread my pylint there w/ the "f"